### ISSUE TITLE
Default Model Licenses

### ISSUE THREAD

**Issue by ankane** [huggingface/transformers#7668]
Hi, thanks for the great library!

I've been trying to compile a list of licenses for the default models and wanted to share in case others were wondering about it. Here's what I have so far:

*Note: table has been updated based on this discussion.*

Task | Model | License | Model Card w/ License
--- | --- | --- | ---
feature-extraction | distilbert-base-cased | Apache-2.0 | ✓ (added)
sentiment-analysis | distilbert-base-uncased-finetuned-sst-2-english | Apache-2.0 | ✓ (added)
ner | dbmdz/bert-large-cased-finetuned-conll03-english | MIT* (added) | 
question-answering | distilbert-base-cased-distilled-squad | Apache-2.0 | ✓ (added)
fill-mask | distilroberta-base | Apache-2.0 | ✓
text-generation | gpt2 | MIT* | ✓
summarization | sshleifer/distilbart-cnn-12-6 | Apache-2.0 | ✓ 
translation, text2text-generation | t5-base | Apache-2.0 | ✓
zero-shot-classification (PyTorch) | facebook/bart-large-mnli | MIT (added) | ✓ (added)
zero-shot-classification (TensorFlow) | roberta-large-mnli |  MIT | ✓
conversational | microsoft/DialoGPT-medium | MIT | ✓

Notes:

- `distil` models without a model card are listed as Apache 2.0 based on this comment: https://github.com/huggingface/transformers/issues/3357#issuecomment-614856396
- `gpt2` was changed from MIT to a custom license earlier this year: [history](https://github.com/openai/gpt-2/commits/master/LICENSE)
- Other `dbmdz` models use MIT (https://github.com/huggingface/transformers/pull/3492), but didn't find info on `dbmdz/bert-large-cased-finetuned-conll03-english`. If the model was fine-tuned from a pretrained BERT model, I imagine it would need to retain Apache 2.0 in addition to how the final model is licensed.

It'd be nice to get clarification on the two models that are missing licenses (and ideally ensure all default models have a clear license going forward).

**Comment by julien-c** [comment]
`facebook/bart-large-mnli` is `mit` like other pretrained models initially released in [fairseq](https://github.com/pytorch/fairseq#license)

For `dbmdz` I'll let @stefan-it chime in, with the caveat that lineage/inheritance of licenses in fine-tuned ML models is (AFAIK) uncharted territory so if you have more info on that subject, please feel free to share it.

Finally, for models where the license isn't indicated in the model card, please feel free to open a PR to add it.

**Comment by alexcombessie** [comment]
Hi @julien-c @ankane,

I am also very interested in clarifying the licenses of default models. In particular, I'd like to know the license of `dbmdz/bert-large-cased-finetuned-conll03-english`.

Cheers,

Alex Combessie

**Comment by ankane** [comment]
Hey @julien-c, thanks for the quick response and `facebook/bart-large-mnli` info. PR submitted.

Re fine-tuning licensing: I don't have any special knowledge here (so will defer further discussion to someone that does), but seems like it may fit the definition of "Derivative Works" in the Apache 2.0 license.

**Comment by stefan-it** [comment]
Hi guys,

sorry for the late reply! I have no strong opinion on that topic, so I would just say that license of our `dbmdz` models will be MIT, because we're usually use this kind of license for both software and our pre-trained LMs :)

**Comment by ankane** [comment]
Great, thanks @stefan-it! That makes it clear that the model is open source :tada:

It'd be good to add a model card with the license. I personally think the most accurate summary of the model license is Apache-2.0 + MIT, unless it wasn't derived from Apache-2.0 work.

**Comment by julien-c** [comment]
On the technical side, just took a look at the code and our YAML parser would support an array of licenses, so feel free to open a PR with 

```
license:
- mit
- apache-2.0
```

On the legal side, 🤷‍♂️

**Comment by ankane** [comment]
Thanks @julien-c, good to know 👍 

Will wait to hear thoughts from @stefan-it.
